bndtools.core/src/bndtools/central/Central.java

```diff
@@ -461,6 +465,9 @@ public static void refreshPlugins() throws Exception {
                         l.modelChanged(p);
                 }
 
+                if (refreshedRepos.size() > 0) {
+                    repositoriesViewRefresher.repositoriesRefreshed();
```

Comment:
@gamerson, are you going to update for my comment https://github.com/bndtools/bndtools/pull/1719#discussion_r119673078?